python/core/fundamental-data-structures/the-fundamental-tuples.md

```diff
@@ -64,35 +57,50 @@ print(fname)
 print(age)
 # 27
 ```
+
 In case of tuples with multiple fields, use the `*` unpacking syntax:
 ```python
 a, *b, c = (1, 2, 3, 4, 5)
 print(b)
 # [2, 3, 4]
 ```
-Tuples are often used for a sequence of values of heterogeneous types (e.g. fields of a database record or columns from a CSV file), while lists are often used for homogeneous items, such as a list of only numbers, or a list of only strings. But this is not a rule; it can be the other way around too.
+
+In this example, we assign the first element of the tuple to `a`, the last element of the tuple to `c`, and everything in-between to `b`.
+
+Tuples are often used for a sequence of values of heterogeneous types (e.g. fields of a database record or columns from a CSV file), while lists are often used for homogeneous items, such as a list of only numbers, or a list of only strings. This is not a rule; it can be the other way around too.
```

Comment:
I find this hella confusing. we should explain what `heterogenous` and `homogenous` mean in this context

Reply:
also we should have a paragraph on "why would I use tuples"

Reply:
hmm okay. this also needs to be split as it already is quite big